test/prepare.bats

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bats
+
```

Comment:
Should include setup and teardown to ensure tests run in isolated directories

Reply:
Let's have an actual talk on this one.  I would like to hear your thoughts.  I spent quite some time yesterday with bats so I would like to leverage any experience you might have here.